changelog/1.17.4.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "1.17.4"
+description: "Released on 04/13/21"
```

Comment:
I may need to fix usage of the `date` script, seems it's only issuing `dd` instead of `dddd`

```suggestion
description: "Released on 04/13/2021"
```